lib/polyfill/media_capabilities.js

```diff
@@ -119,6 +119,13 @@ shaka.polyfill.MediaCapabilities = class {
         if (shaka.util.Platform.isChromecast()) {
           isSupported =
               shaka.polyfill.MediaCapabilities.canCastDisplayType_(videoConfig);
+        } else if (shaka.util.Platform.isTizen()) {
+          let extendedType = videoConfig.contentType;
+          extendedType += `; width=${videoConfig.width}`;
```

Comment:
This 4 properties can be null in some streams, please check if the property is null before append. Thanks!

Reply:
I thought these were all mandatory parameters, but I guess that doesn't mean they can't possibly be null in some type of streams.